### ISSUE TITLE
The target "_CleanProcessGoogleServicesJson" does not exist in the project.

### ISSUE THREAD

**Issue by jrahma** [xamarin/GooglePlayServicesComponents#237]
Hi,

This message keeps showing when I...:

1. Open Visual Studio for Windows
2. Open the solution.
3. Leave the Visual Studio and the solution open.
4. Go to the solution folder on Windows
5. Delete bin and obj folder from my main project.
6. Delete bin and obj folder from my Android project.
7. Go back to Visual Studio
8. Choose Clean Solution

Thanks,
[Jassim](https://www.jassimrahma.net)

**Comment by moljac** [comment]
Can you try restoring nuget packages after step 6 please?

**Comment by Redth** [comment]
No response, closing.  Please open a new issue if you run into more problems on the latest version, after ensuring packages are fully restored.
